### PR TITLE
browser-sdk: add 'grant_device_permission' embed event

### PR DESCRIPTION
**Pull request by fuo213**
### Description

Creates an event for "grant_device_permissions" when a user successfully allows device permissions

Creates an event for "grant_device_permissions" when a user successfully allows device permissions

### Testing

This is a type change, unit tests are included in [PWA4125](https://github.com/whereby/pwa/pull/4125).

### Checklist

-   [ x] My code follows the project's coding standards.
-   [x ] Prefixed the PR title and commit messages with the service or package name
-   [ x] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [ x] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

File: packages/browser-sdk/src/stories/prebuilt-ui.stories.tsx [hunk 1]
```diff
@@ -79,24 +79,33 @@ const WherebyEmbed = ({
...
     const [cameraEnabled, setCameraEnabled] = useState(video);
     const [meetingEnded, setMeetingEnded] = useState(false);
+    const [grantedDevicePermissions, setGrantedDevicePermissions] = useState(false);
 
     useEffect(() => {
         const element = elmRef.current;
 
         element?.addEventListener("camera_toggle", (e) => {
             const cameraEnabled = e.detail.enabled;
+            console.log(cameraEnabled);
```
Comment (thyal) [inline packages/browser-sdk/src/stories/prebuilt-ui.stories.tsx:89]:
```suggestion
```

File: packages/browser-sdk/src/stories/prebuilt-ui.stories.tsx [hunk 2]
```diff
@@ -79,24 +79,33 @@ const WherebyEmbed = ({
...
             setCameraEnabled(cameraEnabled);
         });
 
         element?.addEventListener("meeting_end", () => {
             setMeetingEnded(true);
         });
+
+        element?.addEventListener("grant_device_permission", (e) => {
+            const grantedDevicePermissions = e.detail.granted;
+            console.log(grantedDevicePermissions);
```
Comment (thyal) [inline packages/browser-sdk/src/stories/prebuilt-ui.stories.tsx:99]:
```suggestion
```

**Review by thyal — COMMENTED** [review]
Nice, just remove the console.logs

File: packages/browser-sdk/src/stories/prebuilt-ui.stories.tsx (hunk 1 above)
Comment (fuo213) [reply packages/browser-sdk/src/stories/prebuilt-ui.stories.tsx:89]: haha nice catch. stuff we put in to test with and I just forgot to remove!

**Comment by fuo213** [comment]
@thyal made all the suggested changes, if you could approve that would be awesome :D

**Review by thyal — APPROVED** [review]

**Pull request merged**